project_types/base/project.py

```diff
@@ -226,6 +226,13 @@ def process_project(self):
                     exc_info=True,
                 )
                 self.project.status_message = str(ex)
+            elif isinstance(ex, SoftTimeLimitExceeded):
+                logger.error(
+                    "process_project failed",
```

Comment:
```suggestion
                    "process_project failed due to SoftTimeLimitExceeded",
```
It would be nice to track this in sentry